VisualPinball.Unity/VisualPinball.Unity.Editor/Layers/LayerEditor.cs

```diff
@@ -13,7 +14,12 @@ public class LayerEditor : EditorWindow
 	{
 		private TreeViewState _treeViewState;
 		private TreeView _treeView;
+		private SearchField _searchField;
 
+		private LayerHandler _layerHandler;
+
+		private Rect _toolbarRect { get { return new Rect(20f, 10f, position.width - 40f, 20f); } }
+		private Rect _treeViewRect { get { return new Rect(20, 30, position.width - 40, position.height - 60); } }

```

Comment:
Is that the padding? I would prefer it a bit smaller, it kinda stands out now when the window is docked.

Reply:
Also, less boilerplate would be:
```cs
private Rect _toolbarRect => new Rect(20f, 10f, position.width - 40f, 20f);
private Rect _treeViewRect => new Rect(20, 30, position.width - 40, position.height - 60);
```

Reply:
cannot use this syntax with non static fields.

Reply:
Yes, you can!

What you can't do is make it a field and create the object at class instantiation.